src/static/js/add_tta_match_result.js

```diff
@@ -10,123 +9,221 @@ var tournamentSelector = null;
 
 var playerRankingList = null;
 
+var playerOne = null;
```

Comment:
Ditto, these are quite repeated. Suggest having a single Vue element having two elements in data, e.g.:

Here is an example, although not sure it will work:

https://stackoverflow.com/questions/43239604/multiple-select-and-v-model-with-array-of-object

Reply:
Actually that doesn't seems to be the right example, but my point was maybe we can use v-for to automatically genearate 4 v-select elements without repeating code and html so much.

Reply:
This is an example: https://codepen.io/anon/pen/BvqaBB?editors=1010